### PR TITLE
create flatpages template for about us page

### PR DESCRIPTION
**Pull request by hassanabidpk**
I create Flatpage template for about us page. I couldn't find a way to test it. Maybe write tests in landing site app ? I will do for home page too. #55

### CODE REVIEW

**Comment by hassanabidpk** [comment]
We gotta do migration and add flatpage and site in admin.

File: code4every1site/urls.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
     url(r'^admin/', include(admin.site.urls)),
     url(r'^hacks/', include('hacks.urls', namespace="hacks")),
     url(r'^accounts/', include('accounts.urls', namespace="accounts")),
+    url(r'^about-us/', include('django.contrib.flatpages.urls')),
```
Comment (jinpark) [inline code4every1site/urls.py]: This should probably be just `r^` since you set up the urls in the flatpage app

**Comment by hassanabidpk** [comment]
done

File: code4every1site/urls.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
     url(r'^admin/', include(admin.site.urls)),
     url(r'^hacks/', include('hacks.urls', namespace="hacks")),
     url(r'^accounts/', include('accounts.urls', namespace="accounts")),
+    # url(r'^about-us/', include('django.contrib.flatpages.urls')),
```
Comment (jinpark) [inline code4every1site/urls.py]: I think you need something here to reference the flatpages app

ex.

```
urlpatterns = [
    url(r'^pages/', include('django.contrib.flatpages.urls')),
]
```

**Comment by hassanabidpk** [comment]
I think we don't need it.  when you add flatpage through the admin there is option of url

**Comment by jinpark** [comment]
Let me check it out

**Comment by jinpark** [comment]
image
This is what i see when I go to `admin/flatpages/flatpage/add/` The URL is at the top there

File: code4every1site/templates/flatpages/aboutus.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
```
Comment (jinpark) [inline code4every1site/templates/flatpages/aboutus.html]: We should rename this to `flatpages/default.html`  so we can use it for the rest of the flatpages?

File: code4every1site/templates/flatpages/aboutus.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+{% block title_tag %}
+  <title>{{ flatpage.title }}</title>
+{% endblock title_tag %}
+
+{% block content %}
+<a name="top"></a>
+<div class="android-be-together-section mdl-typography--text-center" style="background: url({% static 'landingsite/images/slide01.jpg' %})
+center 30% no-repeat;height:330px;">
+  <div class="android-font android-slogan">Code for Everyone</div>
+  <div class="android-font android-sub-slogan">About us</div>
```
Comment (jinpark) [inline code4every1site/templates/flatpages/aboutus.html]: If we decide to change this to the default flatpage, we should change `About us` to `{{ flatpage.title }}`

File: code4every1site/urls.py (hunk 2 above)
Comment (jinpark) [inline code4every1site/urls.py]: @hassanabidpk explained that the middleware catches it, very cool :D

**Comment by hassanabidpk** [comment]
Actually `{% static 'landingsite/images/slide01.jpg' %}` doesn't work with you add html contents through admin interface , therefore I think we need it in the individual template for each flatpage we want. (given that we want a separate image)

**Comment by jinpark** [comment]
Yea, we cant use images that way, so we can just remove that part as well and just let people handle it through the admin?

**Comment by hassanabidpk** [comment]
I added template tag following blog post. Check now :+1: s

File: code4every1site/templates/flatpages/aboutus.html [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{% extends "flatpages/default.html" %}
+{% load load_static %}
+
+{% block content %}
+{{ flatpage.content|load_static }}
```
Comment (jinpark) [inline code4every1site/templates/flatpages/aboutus.html]: Why dont we just get rid of this file and move all of this into default.html

File: landingsite/urls.py [hunk 6]
```diff
@@ -8,5 +8,5 @@
     url(r'^contact-us/email/$', views.contact, name='contact'),
     url(r'^contact-us/thanks/',TemplateView.as_view(template_name='landingsite/thanks.html'),name='thanks'),
     url(r'^contact-us/',TemplateView.as_view(template_name='landingsite/contact_us.html'),name='contact-us'),
-    url(r'^about-us/',TemplateView.as_view(template_name='landingsite/about_us.html'),name='about-us'),
+    # url(r'^about-us/',TemplateView.as_view(template_name='landingsite/about_us.html'),name='about-us'),
```
Comment (jinpark) [inline landingsite/urls.py]: We can just delete this instead of commenting it out. :D

**Comment by jinpark** [comment]
After you fix the merge conflict, this should be good to go

**Comment by hassanabidpk** [comment]
fixed the merge conflict :+1: :+1:

**Comment by jinpark** [comment]
Merging, I'll set up the about-us page right after the deploy

**Comment by jinpark** [comment]
Ran migrations, setup about us page, no problems! good job @hassanabidpk

**Pull request merged**